crypto/tls/tls.go

```diff
@@ -69,6 +75,23 @@ func (cfg *ClientConfig) GetTLSConfig() (*tls.Config, error) {
 		config.Certificates = []tls.Certificate{clientCert}
 	}
 
+	if cfg.MinVersion != "" {
+		minVersion, err := cliFlags.TLSVersion(cfg.MinVersion)
+		if err != nil {
+			return nil, errors.Wrapf(err, "failed to set minimum tls version %s", cfg.MinVersion)
```

Comment:
[Nit] For consistency I suggest writing TLS in all caps:
```suggestion
			return nil, errors.Wrapf(err, "failed to set minimum TLS version %s", cfg.MinVersion)
```